docs/docs/user-guide/qa.md

```diff
@@ -0,0 +1,84 @@
+# Quality Assurance
+
+## Overview
+
+Browsertrix Crawler has the capability to "re-crawl" an existing crawl to compare what the browser encountered on a website during crawling against the replay of the crawl WACZ. The WACZ produced by this analysis run includes additional comparison data (stored as WARC `resource` records) for the pages found during crawling against their replay in ReplayWeb.page along several dimensions, including screenshot, extracted text, and page resource comparisons.
+
+!!! note
+
+    QA features described on this page are available in Browsertrix Crawler releases 1.1.0 and later.
+
+
+## Getting started
+
+To be able to run QA on a crawl, we must first have an existing crawl, for example:
```

Comment:
Not mentioned in styleguide, but ([stolen from Google's documentation guidelines](https://developers.google.com/style/pronouns#personal-pronouns)) "we" should refer to the author IE Webrecorder.  I think I need to update the section on "you" statements, which was really about brevity and not a hard rule :\

```suggestion
To be able to run QA on a crawl, you must first have an existing crawl, for example:
```